loleaflet/css/loleaflet.css

```diff
@@ -367,6 +367,12 @@ nav.spreadsheet-color-indicator ~ #sidebar-dock-wrapper {
 	line-height: 1.4;
 }
 
+.loleaflet-annotation-reply-count {
+	margin: 3px;
```

Comment:
Read my other comment and remove the first two property we just need `float: right` here